py/modmath.c

```diff
@@ -371,6 +371,15 @@ STATIC const mp_rom_map_elem_t mp_module_math_globals_table[] = {
     { MP_ROM_QSTR(MP_QSTR___name__), MP_ROM_QSTR(MP_QSTR_math) },
     { MP_ROM_QSTR(MP_QSTR_e), mp_const_float_e },
     { MP_ROM_QSTR(MP_QSTR_pi), mp_const_float_pi },
+    #ifdef mp_const_float_tau
```

Comment:
IMO this should be `#if MICROPY_PY_MATH_CONSTANTS`, to match how other options are included/excluded.

Reply:
Then of course these values need to be defined for obj repr C and D.

Reply:
Ok